daskml/preprocessing/data.py

```diff
@@ -116,3 +120,93 @@ def inverse_transform(self, X, y=None, copy=None):
             return X
         else:
             return _X
+
+
+class QuantileTransformer(skdata.QuantileTransformer):
+    """Transforms features using quantile information.
+
+    This implementation differs from the scikit-learn implementation
+    by using approximate quantiles. The scikit-learn docstring follows.
+    """
+    __doc__ = __doc__ + '\n'.join(
+        skdata.QuantileTransformer.__doc__.split("\n")[1:])
+
+    def _check_inputs(self, X, accept_sparse_negative=False):
+        if isinstance(X, (pd.DataFrame, dd.DataFrame)):
+            X = X.values
+        if isinstance(X, np.ndarray):
+            # TODO: What should the package policy be on inferring
+            # chunksizes?
+            X = da.from_array(X, chunks=10000)
```

Comment:
@mrocklin do you have any thoughts on choosing good, default chunk sizes? Is this even something we should be doing?

The use-case here is someone with (not too big) NumPy arrays, who want to drop this transformer in for the parallelism. Perhaps I should add a `chunks` keyword to the `__init__`, and require that if a NumPy array is used?

Reply:
Perhaps divide the shape by the number of logical cores on the machine?